charts/sam/values.yaml

```diff
@@ -0,0 +1,65 @@
+global:
+  # global.applicationVersion -- What version of the Cromwell application to deploy
+  applicationVersion: latest
+  # global.trustedAddresses -- A map of addresses that will be merged with allowedAddresses.
```

Comment:
Delete these, since trustedAddresses / allowedAddresses aren't applicable for GKE ingress?